site/go.mod

```diff
@@ -3,3 +3,8 @@ module github.com/agones/agones/site
 go 1.21
 
 require gopkg.in/yaml.v2 v2.4.0
+
+require (
+	github.com/google/docsy v0.10.0 // indirect
```

Comment:
Where did this change come from? 🤔

Reply:
> Where did this change come from? 🤔

I think `make site-server` could be the reason. As, his system was having go1.23rc2 version which is unstable. We switched it to go1.21.12
But when we ran `make site-server` nothing has changes and this could be because of either it has used cache or maybe vendor?

Reply:
Weird... when I run `make site-server` nothing changes for me 🤔 🤷🏻‍♂️

Who knows.